cache/models/samples.js

```diff
@@ -0,0 +1,155 @@
+/**
```

Comment:
Lots of API-level stuff in here... I guess I was imagining this more as a replacement for sequelize commands, so the API layer would fork either to execute sequelize cmds or come here for redis cmds... but I guess that would require a larger refactoring or our API layer.
Thoughts?